plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/validation/TextualScenarioValidator.xtend

```diff
@@ -153,24 +149,34 @@ class TextualScenarioValidator extends AbstractTextualScenarioValidator {
 	 */
 	@Check
 	def checkWithExecutionHasDeactivate(Model model) {
-		var messagesWithExecution = newHashMap
+		// keep a list with the messages that contains the withExecution keyword
```

Comment:
update this comment -> the list contains target names, not messages

Reply:
Done